.github/pull_request_template.md

```diff
@@ -28,6 +28,7 @@ Changes proposed in this pull request:
 _This section is for the PR reviewer_
 
 - [ ] PR has an interpretable title with a prefix (refer to [NumPy Development Guide](https://numpy.org/doc/stable/dev/development_workflow.html#writing-the-commit-message))
+  - Example prefixes: `[BUG]`, `[DOC]`, `[ENH]`, `[MAINT]`
```

Comment:
This is a nitpick, but to avoid having a second-level bullet point we could have something like this instead
```md
- [ ] PR has an interpretable title with a prefix (e.g. `[BUG]`, `[DOC]`, `[ENH]`, `[MAINT]`).\
Refer to [NumPy Development Guide](https://numpy.org/doc/stable/dev/development_workflow.html#writing-the-commit-message) for a full list.
- [ ] PR links to GitHub issue with mention `Closes #XXXX`
- [ ] Tests pass
- [ ] Checks pass
```

@michellewang , otherwise, the proposed change are fine too 👍